packages/amplify-provider-awscloudformation/src/upload-appsync-files.ts

```diff
@@ -227,21 +227,17 @@ async function uploadAppSyncFiles(context, resourcesToUpdate, allResources, opti
  * Hashes the project directory into a single value. The same project configuration
  * should return the same hash.
  */
-async function hashDirectory(directory) {
-  const options = {
-    encoding: 'hex',
-    folders: {
-      exclude: ['build'],
-    },
-  };
-
-  const hashResult = await hashElement(directory, options);
-
-  return hashResult.hash;
+export async function hashGQLResource(directory: string, resourceName: string): Promise<string> {
+  const filePaths: string[] = await globby([path.join('**', '*')], {
```

Comment:
glob pattern does not need to use path.join as it is always forward slashes under every OS.

naming: `directory` is `resourceDirectory` ?